### DISCUSSION TITLE
How to use UPPAAL's API to get all Symbolic traces?

### DISCUSSION THREAD

**Discussion by Lqs66** [UPPAALModelChecker/UPPAAL-Meta#119] [Q&A]
I want to get all the Symbolic traces from the initial position to a specific position, does the UPPAAL API support this? If so, how do I do it?
Thank you for your help!

**Comment by yrke** [comment]
Hi 

There is currently no way of doing this, also notice that there might be infinitely many traces :) 
Depending on your goal, you might be able to do some modeling to exclude traces that you have already seen and thereby force uppaal to generate another trace, however, it will properly not be feasible for large models and would require some sort of automation. 

Regards, Kenneth
